habitat_baselines/config/rearrange/gala_kinematic.yaml

```diff
@@ -11,7 +11,7 @@ NUM_PHYSICS_SUBSTEPS: 1
 SAVE_VIDEOS_INTERVAL: -1
 NUM_UPDATES: 60
 NUM_ENVIRONMENTS: 512
-SENSORS: ["DEPTH_SENSOR", "RGB_SENSOR"]
+# SENSORS: ["DEPTH_SENSOR", "RGB_SENSOR", "POINTGOAL_WITH_GPS_COMPASS_SENSOR"]
```

Comment:
removed since it is in habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml